packages/components/src/components/transfer-amount-input/transfer-amount-input.scss

```diff
@@ -0,0 +1,28 @@
+.transfer-amount-input-wrapper {
+    display: flex;
+    flex-direction: column;
+
+    span {
+        font-size: 14px;
+        line-height: 20px;
+    }
+}
+
+.transfer-amount-input-container {
+    position: relative;
+}
+
+.transfer-amount-input {
+    border: none;
+    position: absolute;
+    line-height: 30px;
+    display: flex;
+    flex-shrink: 1;
+    margin: 0;
+
+    input {
+        font-size: 20px;
+        font-weight: 700;
```

Comment:
Wouldn't need these here if we're using `Text` component. We can just pass appropriate props to the `Text` component.

Reply:
@nijil-deriv here we apply properties for input filed. regarding `<span>{label}</span>` i agree with you that we can use Text component. but regarding input i'm not fully understand how you can use Text in this case

Reply:
It seems this is [trickier](https://github.com/binary-com/deriv-app/pull/8442#discussion_r1190580683) 🤔

Reply:
How about using 
`DesktopWrapper` component for  <Text size='xs'>{label}</Text>
and `MobileWrapper` component for  <Text size='xxs'>{label}</Text>

Reply:
I got your query @heorhi-deriv. Yes, for `<input/>` I guess we should stick with these.